app/src/components/data-grid/autocomplete/AsyncAutocompleteDataGridEditCell.tsx

```diff
@@ -42,6 +42,8 @@ export interface IAsyncAutocompleteDataGridEditCell<
    * @memberof IAsyncAutocompleteDataGridEditCell
    */
   error?: boolean;
+
+  renderOption?: (renderProps: any, renderOption: IAutocompleteDataGridOption<ValueType>) => JSX.Element;
```

Comment:
I think a typical `renderXyz` prop should only take `Xyz` as a parameter. It's not certain how the `renderProps` attribute would be used.
```javascript
/**
 * Optional function to render the autocomplete option.
 */
renderOption?: (option: IAutocompleteDataGridOption<ValueType>) => JSX.Element;
```